mars/oscar/backends/ray/context.pyx

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2020 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ...core cimport _Actor, ActorRef
+from ...context cimport BaseActorContext
+from ...errors import ActorNotExist, ActorAlreadyExist
+# TODO(fyrestone): Move lazy_import to oscar
+from ....utils import lazy_import
+
+ray = lazy_import("ray")
+
+
+cdef class _RayActorHandle:
+    cdef object actor_handle
+    cdef object uid
+    cdef object address
+
+
+cdef class RayActorContext(BaseActorContext):
+    cdef public dict actors
+
+    def __init__(self, str address=None, index=0):
+        self.actors = dict()
+
+    async def create_actor(self, object actor_cls, *args, object uid=None, object address=None, **kwargs):
+        cdef _RayActorHandle actor = _RayActorHandle()
+
+        if uid in self.actors:
+            raise ActorAlreadyExist(f'Actor {uid} already exist, cannot create')
+
+        actor_handle = ray.remote(actor_cls).options(name=str(uid)).remote(*args, **kwargs)
+        actor.actor_handle = actor_handle
+        actor.uid = uid
+        actor.address = address
+        self.actors[uid] = actor
+        actor_handle._set_uid.remote(uid)
+        actor_handle._set_address.remote(address)
+        actor_handle.__post_create__.remote()
```

Comment:
Is this method synchronous? Can this call utilize asyncio?

Reply:
The `remote()` call just send the method name and args to remote. This method does not block waiting for the result.

Reply:
``create_actor`` need to return when ``__post_create__`` is finished. Thus we need to wait asynchronously.

Reply:
Got it. I will await the `__post_create__`.